docs/deployment/sso.md

```diff
@@ -66,85 +63,12 @@ SAML2.0 is configured through **Semgrep AppSec Platform**. To set up SSO:
 5. Select the box next to **This SSO supports non-password authentication mechanisms (e.g. MFA, X509, PasswordLessPhoneSignin)** if applicable.
 6. Click **Save** to proceed.
 
-If you are using Google Workspace SAML, see [SAML Single Sign-on with Google Workspace](/docs/kb/semgrep-appsec-platform/saml-google-workspace) for more specific guidance.
-
 If you encounter issues during the setup process, [reach out to support](/docs/support) for assistance.
 
-### Set up SAML SSO with Microsoft Entra ID
-
-<!--
-Semgrep AppSec Platform doesn't have an integration app in Microsoft Entra ID the way it does with Slack and GitHub.
-So, the user has to create a custom app (integration) for SAML SSO.
-We only use SOME steps in the documentation for Azure, so we'll have to make our own guide.
-
-The following references are used:
-
--->
-
-:::info Prerequisites
-* An existing Microsoft Entra ID account.
-* Sufficient permissions within Microsoft Entra ID to create enterprise apps. See [Microsoft Entra ID roles](https://learn.microsoft.com/en-us/azure/active-directory/roles/permissions-reference).
+:::note Admin and org owner accounts
+If you're an admin setting up SSO, Semgrep creates an SSO account for you that is a **Member** by default. To elevate the permissions granted to this account, log in to Semgrep with your admin account using the original login method, then change the role of your newly created SSO account to **Admin**.
```

Comment:
The Linear card suggested a KB article, but I thought it would be more useful here; let me know if you think it should be moved

Reply:
Looks good to me!

Reply:
This isn't actually quite true. It's dependent on a different setting in the deployment, whose default value for more recent orgs is Member, so in that sense it's what most people see. But the result is controlled by the setting at https://semgrep.dev/orgs/armchairlinguist/settings/access/defaults